.github/CONTRIBUTING.md

```diff
@@ -31,7 +31,7 @@ First time contributing to Homebrew? Read our [Code of Conduct](https://github.c
 
 * `brew edit foo` and make edits
 * leave the [`bottle`](http://www.rubydoc.info/github/Homebrew/brew/master/Formula#bottle-class_method) as-is
-* `brew install foo`, `brew test foo`, and `brew audit foo`
+* `brew uninstall foo`, `brew install --build-from-source foo`, `brew test foo`, and `brew audit --strict --online foo`
```

Comment:
Good point from @ilovezfs: let's go for `brew uninstall --force foo`, `brew install --build-from-source foo`